src/ecalc/libraries/libecalc/common/libecalc/core/consumers/consumer_system.py

```diff
@@ -138,134 +138,109 @@ def evaluate(
         operational_settings_used = TimeSeriesInt(
             timesteps=variables_map.time_vector, values=[0] * len(variables_map.time_vector), unit=Unit.NONE
         )
-        operational_settings_results: Dict[int, List[ConsumerModelResult]] = defaultdict(list)
+        operational_settings_results: Dict[int, Dict[str, EcalcModelResult]] = defaultdict(
```

Comment:
We still need to expand on this, as this will fail when temporal operational settings are used.

Reply:
leave as is now, will look at it :)